gb-cpu/src/io_registers.rs

```diff
@@ -34,19 +36,19 @@ impl IORegisters {
 
 impl FileOperation<Area> for IORegisters {
     fn read(&self, _addr: Box<dyn gb_bus::Address<Area>>) -> Result<u8, gb_bus::Error> {
-        Ok(self.enable_mask)
+        Ok(IORegisters::FLAG_MASK | self.enable_mask)
```

Comment:
Won't the u8 resulting of this operation always have the bit 5, 6 and 7 set ? If yes why ?

Reply:
1. Yes, that's the goal of this PR :)
2. because, it seems that unused bitfield are always set to `1` [source](https://mgba-emu.github.io/gbdoc/#ff0f--if-interrupts-asserted)